docs/web3.version.rst

```diff
@@ -27,23 +27,18 @@ The following properties are available on the ``web3.eth`` namespace.
 
 .. py:method:: Version.node(self)
 
+    Method is deprecated in favor of ``web3.clientVersion``.
+
     * Delegates to ``web3_clientVersion`` RPC Method
 
     Returns the current client version.
 
     .. code-block:: python
 
-        >>> web3.version.node
+        >>> web3.clientVersion
```

Comment:
I know this is merged already, but I noticed that there isn't documentation for `web3.clientVersion`. Do you mind adding it when you get a chance? :pray: